pulseapi/profiles/test_views.py

```diff
@@ -433,3 +439,29 @@ def test_invalid_profile_list_filtering_argument(self):
         response = self.client.get(url)
         entriesjson = json.loads(str(response.content, 'utf-8'))
         self.assertEqual(len(entriesjson), 0)
+
+    def test_profile_categories(self):
+        profile_categories_url = reverse('categories_view')
+        response = self.client.get(profile_categories_url)
+        categories = json_to_obj(str(response.content, 'utf-8'))
+        self.assertListEqual(categories.profile_types, [
```

Comment:
let's not hardcode these. Might as well just use the same query you do in the view method.